stable/registry/templates/image-store-pvc.yaml

```diff
@@ -0,0 +1,26 @@
+{{- if .Values.persistence.enabled }}
+kind: PersistentVolumeClaim
+apiVersion: v1
+metadata:
+  name: {{ template "fullname" . }}
+  labels:
+    app: {{ template "fullname" . }}
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+    release: "{{ .Release.Name }}"
+    heritage: "{{ .Release.Service }}"
+  annotations:
+  {{- if .Values.persistence.storageClass }}
+    volume.beta.kubernetes.io/storage-class: {{ .Values.persistence.storageClass | quote }}
+  {{ else }}
+    volume.alpha.kubernetes.io/storage-class: default
+  {{- end }}
+  {{- range $key, $value := .Values.persistence.annotations }}
+    {{ $key }}: {{ $value }}
+  {{- end }}
+spec:
```

Comment:
Do
```
{{- if .Values.persistence.annotations }}
  {{ toYaml .Values.persistence.annotations | indent 4 }}
{{- end }}
```
instead. It's cleaner IMHO. Be sure to add `annotations: {}` to the `persistence:` map in `values.yaml`.